### PR TITLE
Add call back stack and success screen

### PR DESCRIPTION
**Pull request by aledustet**
Why:
----

The call back form should display a success screen for users after they have successfully submitted a call back request from the exposure details screen actions.

This Commit:
----

- Add call back stack to the main navigator
- Move to success screen from the call back form success handler
- Navigate back to the home screen from the success after a call back

Caveat:
----

The success screen returning route is hard-coded to the home screen to avoid complexities of injecting the success route.

Co-authored-by: Devin Jameson<devin@thoughtbot.com>
Co-authored-by: Alejandro Dustet<alejandro@thoughtbot.com>

### CODE REVIEW

File: src/Callback/Success.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { useTranslation } from "react-i18next"
+import { ScrollView, StyleSheet, Image } from "react-native"
+import { useNavigation } from "@react-navigation/native"
+
+import { GlobalText, Button } from "../components"
+import { Typography, Spacing } from "../styles"
+import { Stacks } from "../navigation"
+import { Images } from "../assets"
+
+const Success: FunctionComponent = () => {
```
Comment (devinjameson) [inline src/Callback/Success.tsx:11]: This probably wants `useStatusBarEffect("light-content", Colors.headerBackground)`

**Review by johnschoeman — APPROVED** [review]

File: src/navigation/MainNavigator.tsx [hunk 2]
```diff
@@ -14,6 +15,8 @@ import ActivationStack from "./ActivationStack"
 import SettingsStack from "./SettingsStack"
 import ModalStack from "./ModalStack"
 import Welcome from "../Welcome"
+import CallbackStack from "./CallbackStack"
```
Comment (devinjameson) [inline src/navigation/MainNavigator.tsx:17]:
```suggestion
import CallbackStack from "./CallbackStack"

import { Headers } from "../styles"
```

Comment (devinjameson) [reply src/navigation/MainNavigator.tsx:17]: I wouldn't bother for this commit but usually we put styles and assets in their own group!

File: src/navigation/MainNavigator.tsx [hunk 3]
```diff
@@ -47,6 +51,14 @@ const MainNavigator: FunctionComponent = () => {
                 headerShown: false,
               }}
             />
+            <Stack.Screen
+              name={Stacks.Callback}
+              component={CallbackStack}
+              options={{
+                ...Headers.headerScreenOptions,
```
Comment (devinjameson) [inline src/navigation/MainNavigator.tsx]: At some point we probably want to remove this and add the header within the `Callback` stack, like we do in `Settings`.

**Review by devinjameson — APPROVED** [review]

**Pull request merged**